tests/integration/test_logcollector/test_macos/test_macos_file_status_basic.py

```diff
@@ -0,0 +1,117 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+import re
+import pytest
+import wazuh_testing.logcollector as logcollector
+import wazuh_testing.tools.macos_log.macos_utils as macos_utils
+
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.remote import check_agent_received_message
+from wazuh_testing.tools import LOG_FILE_PATH, WAZUH_PATH
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.tools.monitoring import wait_file
+from wazuh_testing.tools.file import read_json
+from time import sleep
+
+# Marks
+pytestmark = [pytest.mark.darwin, pytest.mark.tier(level=0)]
+
+# Configuration
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+configurations_path = os.path.join(test_data_path, 'wazuh_macos_file_status_basic.yaml')
+
+parameters = [{'ONLY_FUTURE_EVENTS': 'yes'}, {'ONLY_FUTURE_EVENTS': 'no'}]
+metadata = [{'only-future-events': 'yes'}, {'only-future-events': 'no'}]
+
+# Configuration data
+configurations = load_wazuh_configurations(configurations_path, __name__, params=parameters, metadata=metadata)
+configuration_ids = [f'{x["ONLY_FUTURE_EVENTS"]}' for x in parameters]
+
+file_status_path = os.path.join(WAZUH_PATH, 'queue', 'logcollector', 'file_status.json')
+
+macos_log_messages = [
+    {
+        'command': 'logger',
+        'message': 'Logger testing message - file status',
+    }
+]
+
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+
+# Time in seconds to update the file_status.json
+file_status_update_time = 4
+
+local_internal_options = {'logcollector.vcheck_files': str(file_status_update_time)}
+
+
+@pytest.fixture(scope='module')
+def get_local_internal_options():
+    """Get configurations from the module."""
+    return local_internal_options
+
+
+# Fixtures
+@pytest.fixture(scope='module', params=configurations, ids=configuration_ids)
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+
+@pytest.fixture(scope='module')
+def get_connection_configuration():
+    """Get configurations from the module."""
+    return logcollector.DEFAULT_AUTHD_REMOTED_SIMULATOR_CONFIGURATION
+
+
+@pytest.mark.parametrize('macos_message', macos_log_messages)
+def test_macos_file_status_basic(get_local_internal_options, configure_local_internal_options, get_configuration,
+                                 configure_environment, get_connection_configuration, init_authd_remote_simulator,
+                                 macos_message, restart_logcollector):
+
+    """Checks if logcollector stores correctly "macos"-formatted localfile data.
+
+    This test uses logger tool and a custom log to generate an ULS event. The agent is connected to the authd simulator
+    and sends an event to trigger the file_status.json update.
```

Comment:
The custom log is not used in this test, also, is really necessary to send a logger message? In other `file_status` tests this is not used and the status of this file should not be related to sending this event.
```suggestion
    Ensure `file_status.json` file is created and updated properly when macOS log_format is used.
    This test uses logger tool to generate an ULS event. The agent is connected to the authd simulator
    and sends an event to trigger the file_status.json update.
```